alarm/management/commands/price_break_threshold_alarm.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+
+from alarm.models import Coin
+from alarm.binance_utils import connect_binance_socket, close_binance_sockets
+from alarm.utils import check_if_call_needed, get_binance_data_and_update_coin_candle
+
+
+class Command(BaseCommand):
+    help = 'Starts streaming Binance market data'
+
+    def handle(self, *args, **options):
+        # Define the list of currencies and intervals you want to stream
+        currencies = [coin.coin_abbreviation for coin in Coin.objects.all()]
+
+        # Connect to Binance exchange
+        sockets = connect_binance_socket(currencies)
+
+        # Start processing messages
+        try:
+            while True:
+                for socket in sockets:
+                    binance_data = socket.recv()
+
+                    prices = get_binance_data_and_update_coin_candle(
+                        binance_data)
+
+                    check_if_call_needed(prices)
+
+                    if check_if_call_needed:
```

Comment:
I don't see "check_if_call_needed" variable, how you haven't error here?

Reply:
I added variable "threshold_breaks"